2022-09-02 (#169)
rustacean-station/rustacean-station.org: _episodes/interviews/2022-09-02-dmitry-stepanov.md

---
title: "Fyrox with Dmitry Stepanov"

> date: 2022-09-02T17:00:00Z

Reviewer: ```suggestion
date: 2022-09-02T15:45:00Z
```

file: https://audio.rustacean-station.org/file/rustacean-station/2022-09-02-dmitry-stepanov.mp3
duration: "34:18"
length: "32937273"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Dmitry Stepanov](https://github.com/mrDIMAS), creator of [Fyrox](https://fyrox.rs/). Fyrox is a feature-rich, general purpose game engine that is suitable for any kind of games.

Reviewer: ```suggestion
Allen Wyma talks with [Dmitry Stepanov](https://github.com/mrDIMAS), creator of [Fyrox](https://fyrox.rs/). Fyrox is a feature-rich, general purpose game engine built in Rust.
```
